Licensing: Add missing property definitions (#225)
spdx/spdx-3-model: model/Licensing/Properties/name.md

## Summary

A name identifies the full name of a License.

## Description

A name contains the full name of a License, preferably using the title found
in the applicable license text or file, or as otherwise specified by the

> License's author or steward..

Reviewer: ```suggestion
License's author or steward.
```

> Where no title is specified, a name used by another well-known source or list
> of licenses (such as OSI or Fedora) may also be preferable.

Reviewer: ```suggestion
When no such title is specified, using a name from another well-known source or list
of licenses (such as OSI or Fedora) is suggested.
```
